mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfigurationWritable.java

```diff
@@ -168,6 +174,36 @@ public int heartbeatTimeoutMs() {
         return this.heartbeatTimeoutMs;
     }
 
+    @Override
+    public long heartbeatRetryInitialDelayMs() {
```

Comment:
Doesn't need to be for this PR: seems like we can consider separate worker/agent config vs worker runtime (for the actual job worker) config. These ones don't need to be passed into the job worker I guess.